src/main/resources/application.properties

```diff
@@ -37,21 +37,21 @@ spring.servlet.multipart.max-file-size = 128KB
 spring.servlet.multipart.max-request-size = 128KB
 management.endpoints.web.exposure.include = *
 
-spring.mail.host = smtp.gmail.com
-spring.mail.port = 587
+
 
 # TODO use environment variable for security properties
 spring.mail.username = ${AMPPD_USERNAME:username}
```

Comment:
as per our conversation in slack, i'd suggest change spring.mail.username to use AMPPD_admin instead of USERNAME. as the latter could be any generic value we set up in User table for testing AMP, a diff value than admin

Reply:
done